pom.xml

```diff
@@ -520,6 +566,7 @@
         <org.json.wso2.version>2.0.0.wso2v1</org.json.wso2.version>
         <axiom-api.version>1.2.15</axiom-api.version>
         <axiom-impl.version>1.2.15</axiom-impl.version>
+        <axis2.wso2.version>1.6.1.wso2v10</axis2.wso2.version>
```

Comment:
I can move this to the feature repo. Since the existing code uses axis2 + since it's an external feature, we can keep axis2